lib/python/qmk/constants.py

```diff
@@ -189,3 +189,126 @@
 #
 ################################################################################
 '''
+
+LICENSE_TEXTS = [
+    (
+        'GPL-2.0-or-later', [
+            """\
+        This program is free software; you can redistribute it and/or
+        modify it under the terms of the GNU General Public License
+        as published by the Free Software Foundation; either version 2
+        of the License, or (at your option) any later version.
+        """, """\
+        This program is free software; you can redistribute it and/or
+        modify it under the terms of the GNU General Public License
+        as published by the Free Software Foundation; either version 2
+        of the License, or any later version.
+        """
+        ]
+    ),
+    ('GPL-2.0-only', ["""\
```

Comment:
This inconsistency in formatting bugs me.

Reply:
Same, but this is what the formatter spits out. Annoying.

Reply:
We could change the data structure, but even that does not make the formatting fully consistent between the cases with single and multiple values:
```python
LICENSE_TEXTS = {
    'GPL-2.0-or-later': [
        """\
        This program is free software; you can redistribute it and/or
        modify it under the terms of the GNU General Public License
        as published by the Free Software Foundation; either version 2
        of the License, or (at your option) any later version.
        """, """\
        This program is free software; you can redistribute it and/or
        modify it under the terms of the GNU General Public License
        as published by the Free Software Foundation; either version 2
        of the License, or any later version.
        """
    ],
    'GPL-2.0-only': ["""\
        This program is free software; you can redistribute it and/or
        modify it under the terms of the GNU General Public License as
        published by the Free Software Foundation; version 2.
        """],
```
If all else fails, you could slap `# yapf:disable` after the final closing bracket and format the value manually.

Reply:
I don't feel a strong enough need to fight the formatter. This is something that'll be touched only once every blue moon.